src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -875,6 +875,7 @@ Object AsyncRPCOperation_sendmany::perform_joinsplit(
             {info.vjsout[0], info.vjsout[1]};
     boost::array<size_t, ZC_NUM_JS_INPUTS> inputMap;
     boost::array<size_t, ZC_NUM_JS_OUTPUTS> outputMap;
+    std::function<int(int)> emptyFunc;
```

Comment:
I would suggest just reverting 2eeb6b (but keeping the new test). That seems like a safe fix, since it is reverting to code that we were using on testnet for several releases.